### PR TITLE
Enable etcd for events only in large clusters

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #16424

@davidopp @brendandburns @gmarek

### CODE REVIEW

**Comment by wojtek-t** [comment]
@brendandburns - can you please take a look (or reassign?)

**Comment by wojtek-t** [comment]
@brendandburns @davidopp - can one of you please take a look?

File: cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest [hunk 1]
```diff
@@ -41,7 +41,11 @@
 {% endif -%}
 
 {% set etcd_servers = "--etcd-servers=http://127.0.0.1:4001" -%}
-{% set etcd_servers_overrides = "--etcd-servers-overrides=/events#http://127.0.0.1:4002" -%}
+{% set etcd_servers_overrides = "" -%}
+# If there are more than 50 nodes, there is a dedicated etcd instance fo events.
```
Comment (davidopp) [inline cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest]: s/fo/for/

**Comment by davidopp** [comment]
Thanks for doing this. Can you add documentation to
https://github.com/kubernetes/kubernetes/blob/master/docs/admin/cluster-large.md
?

File: cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest (hunk 1 above)
Comment (wojtek-t) [inline cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest]: done

**Comment by wojtek-t** [comment]
@davidopp - added a section to the doc you mentioned.

PTAL

**Comment by brendandburns** [comment]
LGTM.

**Comment by davidopp** [comment]
LGTM

**Pull request merged**